app/src/test/java/com/android/unio/model/association/AssociationRepositoryFirestoreTest.kt

```diff
@@ -185,13 +206,24 @@ class AssociationRepositoryFirestoreTest {
           assertEquals(association1.name, associations[0].name)
           assertEquals(association1.fullName, associations[0].fullName)
           assertEquals(association1.description, associations[0].description)
-          assertEquals(association1.members.uids, associations[0].members.uids)
+          println("Ok")
+          println(associations[0])
+          println(association1)
```

Comment:
This shouldn't be left before merging to main

Reply:
woupss